hs_tracking/views.py

```diff
@@ -0,0 +1,63 @@
+import csv
+from cStringIO import StringIO
+
+from django.views.generic import TemplateView, View
+from django.utils.decorators import method_decorator
+from django.contrib.auth.decorators import user_passes_test
+from django.http import HttpResponse
+
+from . import models as hs_tracking
+
+
+class UseTrackingView(TemplateView):
+    template_name='hs_tracking/tracking.html'
+
+    @method_decorator(user_passes_test(lambda u: u.is_staff))
+    def dispatch(self, *args, **kwargs):
+        return super(UseTrackingView, self).dispatch(*args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        return {}
+
+
+class VisitorProfileReport(TemplateView):
+
+    @method_decorator(user_passes_test(lambda u: u.is_staff))
+    def dispatch(self, *args, **kwargs):
+        return super(VisitorProfileReport, self).dispatch(*args, **kwargs)
+
+    def get(self, request, **kwargs):
+        """Download a CSV report of use tracking data."""
+
+        f = StringIO()
+        w = csv.writer(f)
+        w.writerow(hs_tracking.VISITOR_FIELDS)
+        visitors = hs_tracking.Visitor.objects.all()
+
+        for v in visitors:
+            info = v.export_visitor_information()
+            row = [info[field] for field in hs_tracking.VISITOR_FIELDS]
+            w.writerow(row)
+        f.seek(0)
```

Comment:
@ironfroggy Just tried on beta for this branch @mjstealey just deployed. I found there are 6 "empty" lines in profile report in the format like "3,,,,,,,,,," I am wondering whether some filtering needs to be done to make sure to only write out the rows when contents are not empty?